modules/ximgproc/CMakeLists.txt

```diff
@@ -1,2 +1,5 @@
 set(the_description "Extended image processing module. It includes edge-aware filters and etc.")
 ocv_define_module(ximgproc opencv_core opencv_imgproc opencv_calib3d opencv_imgcodecs WRAP python)
+
+file(COPY samples/peilin_plane.png DESTINATION ${OpenCV_BINARY_DIR}/bin)
```

Comment:
These files are needed only in samples. If `BUILD_EXAMPLES=OFF` in cmake the files become useless at all. I suggest to delete copy commands at all, because such a style of storing samples data is not used in OpenCV.